CGAL_ImageIO/src/CGAL_ImageIO/gis_impl.h

```diff
@@ -143,7 +143,7 @@ int writeGis( char *name, _image* im) {
 		if ( j<n && i<size ) sprintf( str+strlen(str), " " );
 	      }
 	      sprintf( str+strlen(str), "\n" );
-	      res = ImageIO_write( im, str, strlen( str )  );
+	      res = static_cast<int>(ImageIO_write( im, str, strlen( str )));
 	      if ( res  <= 0 ) {
```

Comment:
`writeGisHeader` should return a `bool`, instead of the two value `-1` and `1`.
Then:
```C++
  if (! writeGisHeader(im)) {
     fprintf(stderr, "writeGis: error: unable to write header of \'%s\'\n",
 	    outputName);
     if ( outputName != NULL ) ImageIO_free( outputName );
     ImageIO_close( im );
     im->fd = NULL;
     im->openMode = OM_CLOSE;
     return( -1 );
  }
```